fast.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
I assume this file was left by mistake

Reply:
for sure, rushed to push to show. Needs a bit of clean up still. Wasn't sure it was going to pass the soak at first.